docs/src/reference/contributing_to_sway.md

```diff
@@ -0,0 +1,101 @@
+# Contributing To Sway
+
+Thanks for your interest in contributing to Sway! This document outlines the process for installing and setting up the Sway toolchain for development, as well as some conventions on contributing to Sway.
+
+If you run into any difficulties getting started, you can always ask questions on our [Discord](https://discord.gg/xfpK4Pe).
+
+## Building and setting up a development workspace
+
+See the [introduction](../introduction/index.md) section for instructions on installing and setting up the Sway toolchain..
+
+## Getting the repository
+
+1. Visit the [Sway](https://github.com/FuelLabs/sway) repo and fork the project.
+2. Then clone your forked copy to your local machine and get to work.
+
+```sh
+git clone https://github.com/FuelLabs/sway
+cd sway
+```
+
+## Building and testing
+
+The following steps will run the sway test suite and ensure that everything is set up correctly.
+
+First, open a new terminal and start `fuel-core` with:
```

Comment:
Maybe prior to this we should have a section on installing `fuel-core` along with a brief sentence explaining why it's required to run the tests? Or perhaps provide a link if we already have a guide on `fuel-core` setup elsewhere?

Reply:
The Sway Book should already talk about installing `fuel-core`, so a link should be sufficient.

Reply:
The section above, `Building and setting up a development workspace` already has a link to the sway book's [introduction](https://fuellabs.github.io/sway/latest/introduction/index.html) which _should_ be sufficient?